package.json

```diff
@@ -214,15 +214,15 @@
                 "command": "language-julia.chooseModule",
                 "title": "Julia: Select Current Module"
             },
-            {
-                "command": "language-julia.debugEditorContents",
-                "title": "Debug Julia File",
-                "icon": "$(debug-alt)"
-            },
             {
                 "command": "language-julia.runEditorContents",
-                "title": "Run Julia File",
+                "title": "Julia: Run File",
```

Comment:
I think it's better to have a consistent naming for our commands, but let me know if you have any thought on having a different format for these commands, @davidanthoff